fetch.bs

```diff
@@ -7417,8 +7479,14 @@ method steps are:
    with <var>p</var>, <var>request</var>, and <var>responseObject</var>, and terminate these
    substeps.
 
-   <li><p>If <var>response</var> is a <a>network error</a>, then <a for=/>reject</a> <var>p</var>
-   with a {{TypeError}} and terminate these substeps.
+   <li><p>If <var>response</var> is a <a>network error</a>, then:
+   <ol>
+    <li><p><a for=/>Reject</a> <var>p</var> with a {{TypeError}}.
+
+    <li><p>Call <var>handleFetchDone</var> with <var>response</var>.
+
+    <li><p>terminate these substeps.
+   </ol>
```

Comment:
I think this is wrong. At least, in my mental model processResponseDone should run when you get a network error.